Sources/Rendering/OpenGL/PolyDataMapper/index.js

```diff
@@ -2018,7 +2022,7 @@ export function extend(publicAPI, model, initialValues = {}) {
   }
 
   // Build VTK API
-  macro.setGet(publicAPI, model, ['context']);
+  macro.setGet(publicAPI, model, ['context', 'coordShiftScaleEnabled']);
```

Comment:
model.coordShiftScaleEnabled is not initialized on purpose ? 
Therefore false by default ?
Why would coordShiftScaleEnabled have a different value between the API specific mapper (vtkOpenGLPolyDataMapper) and the original mapper (vtkPolyDataMapper)

Reply:
This should be added in the documentation.